Makefile

```diff
@@ -59,6 +59,9 @@ migrate: compose_build
 createuser: compose_build
 	docker-compose run app python manage.py createsuperuser
 
+load_initial_data: compose_build
```

Comment:
`initial data` is a pretty broad descriptor, I think something like 'load-locale-data' would be more descriptive.

Reply:
It's not just locales. It's countries too. And in the future there might be more. 

`load_base_data` better?

Reply:
How about `load-locales-countries`?